src/graph_builder.js

```diff
@@ -0,0 +1,27 @@
+import Graph from './rdf/graph'
+import Statement from './rdf/statement'
+/**
```

Comment:
Would like a single blank line after imports. (Would be ideal if eslint did this for us. I will look into that.)